src/main/java/com/skilld/kubernetes/JobConfiguration.java

```diff
@@ -95,6 +99,14 @@ public Integer getParallelism() {
 		return labels;
 	}
 
+	public String getPersistentVolumeName() { return persistentVolumeName; }
```

Comment:
we could have a single "getPersistentVolume"  to be consistent with the setter and have it returns an array of string or a string map for instance; what do you think ?